packages/matchbox/src/components/Panel/Footer.js

```diff
@@ -1,44 +1,48 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Grid } from '../Grid';
-import styles from './Panel.module.scss';
+import styled from 'styled-components';
+import { footer, left, right } from './styles';
 
-class Footer extends Component {
+const OuterWrapper = styled('div')`
+  ${footer}
+`;
 
-  static displayName = 'Panel.Footer';
+const LeftColumn = styled('div')`
+  ${left}
+`;
 
-  static propTypes = {
-    /**
-      * Left aligned content
-      */
-    left: PropTypes.node,
+const RightColumn = styled('div')`
+  ${right}
+`;
 
-    /**
-      * Right aligned content
-      */
-    right: PropTypes.node
-  };
+function Footer(props) {
+  const {
+    // Left aligned content
+    left,
+    // Right aligned content
+    right,
+    ...rest
+  } = props;
 
-  render() {
-    const { left, right, ...rest } = this.props;
-
-    return (
-      <div className={styles.Footer} {...rest}>
-        <Grid>
-          <Grid.Column xs={6}>
-            <div className={styles.Left}>
-              {left}
-            </div>
-          </Grid.Column>
-          <Grid.Column xs={6}>
-            <div className={styles.Right}>
-              {right}
-            </div>
-          </Grid.Column>
-        </Grid>
-      </div>
-    );
-  }
+  return (
+    <OuterWrapper {...rest}>
+      <Grid>
```

Comment:
Let's not use `Grid` here, I think we're going to be deprecating this component pretty soon. You can achieve the same thing with `Box` and some flex properties